pom.xml

```diff
@@ -111,8 +111,8 @@
         <dependency.json-smart.version>2.4.10</dependency.json-smart.version>
         <dependency.jakarta-rpc-api.version>1.1.4</dependency.jakarta-rpc-api.version>
 
-        <alfresco.googledrive.version>3.4.0</alfresco.googledrive.version>
-        <alfresco.aos-module.version>1.6.0</alfresco.aos-module.version>
+        <alfresco.googledrive.version>3.4.1</alfresco.googledrive.version>
+        <alfresco.aos-module.version>1.6.1</alfresco.aos-module.version>
```

Comment:
I can see that **[`googledrive:3.4.1`](https://github.com/Alfresco/googledrive/compare/3.4.0...3.4.1)** relies on Java 17, the same applies to **[`aos-module:1.6.1`](https://github.com/Alfresco/alfresco-aos-module/compare/1.6.0...1.6.1)**. Were we using these versions for testing purposes or are we actually going to merge them?

Reply:
Yes we want to merge these to really "see" when CI becomes green when Java 11 compatible versions are integrated.